docs/api.md

```diff
@@ -0,0 +1,39 @@
+# Hub Router APIs
+
+### Invitation API - HTTP GET /didcomm/invitation
+Returns hub-router DIDComm [Out-Of-Band invitation](https://github.com/hyperledger/aries-rfcs/tree/master/features/0434-outofband#invitation-httpsdidcommorgout-of-bandverinvitation).
+
+#### Response 
+``` json
+{
+   "invitation":{ <oob_invitation> }
+}
+```
+
+##### Sample Response
+``` json
+{
+   "invitation":{
+      "@id":"4fb5bb1d-705b-4be2-9fe3-0a406232ac8f",
+      "@type":"https://didcomm.org/oob-invitation/1.0/invitation",
+      "label":"hub-router",
+      "service":[
+         {
+            "ID":"58fe0316-ea25-4db4-b4b2-946a9b75cb3e",
+            "Type":"did-communication",
+            "Priority":0,
+            "RecipientKeys":[
+               "8XQawExAm8s2N1U9i4zBWEUmeqBDW3rfDLnqoSn92acc"
+            ],
+            "RoutingKeys":null,
```

Comment:
Should you omit routing keys if empty - not sure about spec :)

Reply:
[Removed](https://github.com/trustbloc/hub-router/compare/1b2fa7cdbdf6105594a6bd9f7f8b15431cdfcb32..2e9d8f798132690b012126918748d9e0ae6b6570).

Reply:
Thanks, I missed to validate the sample response :)